x-pack/solutions/security/plugins/elastic_assistant/server/ai_assistant_service/index.ts

```diff
@@ -121,6 +133,8 @@ export class AIAssistantService {
   private alertSummaryDataStream: DataStreamSpacesAdapter;
   private anonymizationFieldsDataStream: DataStreamSpacesAdapter;
   private defendInsightsDataStream: DataStreamSpacesAdapter;
+  private checkpointsDataStream: IndexPatternAdapter;
```

Comment:
Why don't we use data streams for these two resources? We already have all setup and install functionality.